### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by CorneliusAkanbi1**

### CODE REVIEW

**Review by VictorIbikunle — COMMENTED** [review]
Could emphasize more on the guidelines for effective code reviews and make references to an article or blog as such to back up your statements, other than that it looks good.

File: Code Reviews/CodeText.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+1. introduction to Code Reviews:
```
Comment (danieljagun) [inline Code Reviews/CodeText.md]: Capital I

File: Code Reviews/CodeText.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+code reviews involve inspecting code changes made by one team member before they are merged into the main codebase. This process allows for identifying bugs, improving code quality, and ensuring consistency across the codebase. Code reviews also facilitate knowledge sharing among team members and provide an opportunity for mentorship and learning.
```
Comment (danieljagun) [inline Code Reviews/CodeText.md]: Capital C

File: Code Reviews/CodeText.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+2. guidelines for Effective Code Reviews:
```
Comment (danieljagun) [inline Code Reviews/CodeText.md]: Capital G

File: Code Reviews/CodeText.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+Provide Constructive Feedback: Offer specific and actionable feedback, focusing on improving the code rather than criticizing the developer.
+
+
+3. Common Pitfalls to Avoid  :
+Nitpicking: Avoid overly nitpicking minor style issues that don't significantly impact code quality or functionality.
+Lack of Ownership: Ensure that both the author and reviewer take ownership of the code changes and follow through on addressing feedback.
+Ignoring Feedback: don't dismiss feedback without consideration; take the time to understand and address reviewer comments.
+Rushing Through Reviews: Avoid rushing through code reviews to meet deadlines, as this can lead to missed defects and lower-quality code.
+
+4. Reference Articles:
```
Comment (danieljagun) [inline Code Reviews/CodeText.md]: For referencing section, could maybe just include the link only or link / author?

**Review by danieljagun — APPROVED** [review]

**Review by VictorIbikunle — APPROVED** [review]

**Pull request merged**